war/src/main/less/form/search-bar.less

```diff
@@ -0,0 +1,208 @@
+.jenkins-search {
+  --search-bar-height: 2.1875rem;
+
+  position: relative;
+  max-width: 420px;
+
+  &__input {
+    appearance: none;
+    background: var(--item-background--hover);
+    border: none;
+    outline: none;
+    border-radius: 0.625rem;
+    width: 100%;
+    margin: 0;
+    padding: 0 0.5rem 0 var(--search-bar-height);
+    line-height: 1;
+    box-shadow: var(--form-input-glow);
+    transition: var(--standard-transition);
+    font-weight: 500;
+    height: var(--search-bar-height);
+
+    &::placeholder {
+      color: var(--text-color-secondary);
+    }
+
+    // Safari adds unwanted padding - let's remove it
+    &::-webkit-search-decoration {
+      -webkit-appearance: none;
+    }
+
+    &::-webkit-search-cancel-button {
+      -webkit-appearance: none;
+      height: 45%;
+      min-height: 1.1rem;
+      aspect-ratio: 1;
+      margin-right: 0.2rem;
+      background: currentColor;
+      mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 512 512'%3E%3Cpath d='M256 48C141.31 48 48 141.31 48 256s93.31 208 208 208 208-93.31 208-208S370.69 48 256 48zm75.31 260.69a16 16 0 11-22.62 22.62L256 278.63l-52.69 52.68a16 16 0 01-22.62-22.62L233.37 256l-52.68-52.69a16 16 0 0122.62-22.62L256 233.37l52.69-52.68a16 16 0 0122.62 22.62L278.63 256z'/%3E%3C/svg%3E");
+      mask-size: contain;
+      mask-repeat: no-repeat;
+      opacity: 0;
+      pointer-events: none;
+      transform: scale(0.8);
+      transition: var(--standard-transition);
+      cursor: pointer;
+
+      &:hover {
+        opacity: 0.75 !important;
+      }
+
+      &:active {
+        opacity: 1 !important;
+      }
+    }
+
+    &:hover {
+      background: var(--item-background--active);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+      background: var(--item-background--active);
+      box-shadow: var(--form-input-glow--focus);
```

Comment:
focus state seems off compared to other inputs:

<img width="352" alt="image" src="https://user-images.githubusercontent.com/21194782/188743152-8a5c638f-3535-49d1-b4d4-fea27d82dd00.png">

vs

<img width="950" alt="image" src="https://user-images.githubusercontent.com/21194782/188743250-834ba7dc-d77a-448a-9952-d5fbcb922514.png">

Reply:
header search has some extra styles that make it look consistent:

<img width="467" alt="image" src="https://user-images.githubusercontent.com/21194782/188743827-516815d9-863b-45e8-919c-800bbcfd2c5b.png">

Reply:
<img width="351" alt="Screenshot 2022-09-07 at 00 15 04" src="https://user-images.githubusercontent.com/43062514/188756206-52ec02f5-ebe9-428d-9d24-334087e3c718.png">

<img width="754" alt="Screenshot 2022-09-07 at 00 15 13" src="https://user-images.githubusercontent.com/43062514/188756220-0c0cc284-974b-40e2-8b5c-7048b1735486.png">

Updated it 👍